service/src/main/java/org/apache/griffin/core/job/JobInstance.java

```diff
@@ -281,7 +286,8 @@ private void createJobInstance(Map<String, Object> confMap)
             throw new GriffinException.ConflictException
                     (QUARTZ_JOB_ALREADY_EXIST);
         }
-        saveJobInstance(jobName, groupName);
+        String triggerName = (String) confMap.get(TRIGGER_KEY);
```

Comment:
why trigger _key_ is named on bean object as trigger _name_?

Reply:
Fixed